lib/tessel/commands.js

```diff
@@ -7,6 +7,9 @@ module.exports.scanWiFi = function() {
 module.exports.stopRunningScript = function() {
   return ['/etc/init.d/tessel-app', 'stop'];
 };
+module.exports.disablePushedScript = function() {
+  return ['/etc/init.d/tessel-app', 'disable'];
```

Comment:
@rwaldron the name is `erase` for legacy reasons. On Tessel 1, we literally just erased the code that was in flash to clear out a `tessel push`. I'm open to other suggestions. `tessel unpush`?